parm/config/gfs/config.anal

```diff
@@ -19,12 +19,8 @@ if [[ "${CDUMP}" = "gfs" ]] ; then
   export DIAG_TARBALL="YES"
 fi
 
-export npe_gsi=${npe_anal}
-
-if [[ "${CDUMP}" == "gfs" ]] ; then
-  export npe_gsi=${npe_anal_gfs}
-  export nth_anal=${nth_anal_gfs}
-fi
+npe_var="npe_anal_${RUN//enkf}"
```

Comment:
I think we are using only one slash everywhere else, since `$RUN` will only have `enkf` once anyway.
```suggestion
npe_var="npe_anal_${RUN/enkf}"
```